project/ProjectSettings.scala

```diff
@@ -32,7 +32,7 @@ object ProjectSettings {
     Compile / packageDoc / publishArtifact := false,
     Compile / doc / sources := Seq.empty,
     Compile / doc := target.map(_ / "none").value,
-    scalaVersion := "2.13.12",
+    scalaVersion := "2.13.13",
```

Comment:
Is it strictly necessary to update the Scala version for this change?

Reply:
It is not absolutly necessary, it could be done in another PR probably, however I recommend to do it in par with the Play upgrade for now, because we already test Play 3.0.2 with Scala 2.13.13, so this should be safe. Same with the sbt upgrade.

In general I recommend you to upgrade to the latest Scala versions as soon as they come out. I have seen that sometimes you lack behind a couple of (patch) versions in various of your repos. I get the "never change a running system" thing, however usually the Scala upgrades should apply safely and sometimes they bring fixes and small new features and also make life easier to upgrade to newer Play (patch) releases later or even JVM (patch) releases.

Reply:
I think it is a good practice to upgrade to the latest version of scala environment together, rather than trying to go with atomic PRs, because you benefits from latest bugs fixes.

Reply:
I have now crafted this https://github.com/guardian/recommendations/pull/164